rules/solid/tests/Rector/If_/ChangeAndIfToEarlyReturnRector/Fixture/equals.php.inc

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Rector\SOLID\Tests\Rector\If_\ChangeAndIfToEarlyReturnRector\Fixture;
+
+class EqualsClass
+{
+    public function canDrive(Car $car)
+    {
+        if ($car->hasWheels === true && $car->hasFuel === true) {
```

Comment:
Could you also add test for 3 items?

```php
$value === true && $value === true && $value === true 
```

Reply:
At the moment, the case with 3 items is skipped, there is a fixture for this. More than two items will make rule a bit more complicated I think. I wanted to keep the PR as small as possible.

Reply:
I see